lib/matplotlib/pyplot.py

```diff
@@ -1552,7 +1552,7 @@ def yscale(scale, **kwargs):
     gca().set_yscale(scale, **kwargs)
 
 
-def xticks(*args, **kwargs):
+def xticks(locs=None, labels=None, **kwargs):
```

Comment:
What are the names of these arguments in other similar parts of the API? (i.e. are "locs" and "labels" what we want to call these?)

Reply:
Actually the argument for `Axes.set_xticks` and `Axis.set_ticks` is named `ticks`, so let's stay consistent and change the docstring accordingly (remember that argument names effectively become part of the public API).
The argument for `Axes.set_xticklabels` is named `labels` but the one for `Axis.set_ticklabels` is `ticklabels`, which is unfortunate but I think `labels` is fine.